modules/config.xqm

```diff
@@ -755,6 +755,31 @@ declare variable $config:PUBLICATIONS :=
               else ()
           }
         },
+        "search": map{
+            "publication-last-modified": 
+                function() {
+                    let $dates := (
```

Comment:
I have a hard time accepting "search" as a publication and would like to see the last-modified header to be set elsewhere.

Reply:
You are sorting a sequence of `xs:dateTime`, that is why I am asking.

Reply:
What do you think about refactoring the search function to
```
head(
  for $collection in $collections 
  let $date := config:last-modified-from-repo-xml($collection)()
  order by $date descending
  return $date
)
```

Reply:
Instead of sorting, we could just use `fn:max()` on the sequence of dates.